### ISSUE TITLE
-.slice: Failed to set 'memory.limit_in_bytes' attribute on '/' to '-1': Invalid argument

### ISSUE THREAD

**Issue by mbiebl** [systemd/systemd#10969]
**systemd version the issue has been seen with**

v239-2818-g66e383401

**Used distribution**

Debian sid

During boot or on a ```daemon-reexec``` I see the following error messages in the journal:
```
Nov 28 09:00:27 pluto systemd[1]: -.slice: Failed to set 'memory.limit_in_bytes' attribute on '/' to '-1': Invalid argument
Nov 28 09:00:27 pluto systemd[1]: -.slice: Failed to set 'cpu.cfs_quota_us' attribute on '/' to '-1': Invalid argument
Nov 28 09:00:27 pluto systemd[1]: -.slice: Failed to set 'cpu.cfs_period_us' attribute on '/' to '100000': Invalid argument
```

**Comment by poettering** [comment]
Hmm, can't make sense of `v239-2818-g66e383401`... There is not commit g66e383401 in our tree...

Is this pre or post 52fecf20b9269f4675226a8e600cac0bad0539ff?

**Comment by mbiebl** [comment]
It's commit 66e383401

**Comment by mbiebl** [comment]
So yes, post 52fecf2

**Comment by mbiebl** [comment]
```
$ uname -a
Linux pluto 4.18.0-2-amd64 #1 SMP Debian 4.18.10-2 (2018-11-02) x86_64 GNU/Linux

$ grep -E "(BPF|CGROUP)" /boot/config-4.18.0-2-amd64 
CONFIG_CGROUPS=y
CONFIG_BLK_CGROUP=y
# CONFIG_DEBUG_BLK_CGROUP is not set
CONFIG_CGROUP_WRITEBACK=y
CONFIG_CGROUP_SCHED=y
CONFIG_CGROUP_PIDS=y
# CONFIG_CGROUP_RDMA is not set
CONFIG_CGROUP_FREEZER=y
# CONFIG_CGROUP_HUGETLB is not set
CONFIG_CGROUP_DEVICE=y
CONFIG_CGROUP_CPUACCT=y
CONFIG_CGROUP_PERF=y
CONFIG_CGROUP_BPF=y
# CONFIG_CGROUP_DEBUG is not set
CONFIG_SOCK_CGROUP_DATA=y
CONFIG_BPF=y
CONFIG_BPF_SYSCALL=y
# CONFIG_BPF_JIT_ALWAYS_ON is not set
CONFIG_IPV6_SEG6_BPF=y
CONFIG_NETFILTER_XT_MATCH_BPF=m
CONFIG_NETFILTER_XT_MATCH_CGROUP=m
# CONFIG_BPFILTER is not set
CONFIG_NET_CLS_CGROUP=m
CONFIG_NET_CLS_BPF=m
CONFIG_NET_ACT_BPF=m
CONFIG_CGROUP_NET_PRIO=y
CONFIG_CGROUP_NET_CLASSID=y
CONFIG_BPF_JIT=y
# CONFIG_BPF_STREAM_PARSER is not set
CONFIG_LWTUNNEL_BPF=y
CONFIG_HAVE_EBPF_JIT=y
CONFIG_BPF_EVENTS=y
# CONFIG_BPF_KPROBE_OVERRIDE is not set
CONFIG_TEST_BPF=m
```

**Comment by poettering** [comment]
Hmm, that's I guess what I get if I don't test cgroupsv1 setups sufficiently anymore... ;-)

**Comment by mbiebl** [comment]
```
│ ├─/sys/fs/cgroup                    tmpfs          tmpfs      ro,nosuid,nodev,noexec,mode=755
│ │ ├─/sys/fs/cgroup/unified          cgroup2        cgroup2    rw,nosuid,nodev,noexec,relatime
│ │ ├─/sys/fs/cgroup/systemd          cgroup         cgroup     rw,nosuid,nodev,noexec,relatime,xattr,name=systemd
│ │ ├─/sys/fs/cgroup/cpuset           cgroup         cgroup     rw,nosuid,nodev,noexec,relatime,cpuset,clone_childr
│ │ ├─/sys/fs/cgroup/net_cls,net_prio cgroup         cgroup     rw,nosuid,nodev,noexec,relatime,net_cls,net_prio
│ │ ├─/sys/fs/cgroup/freezer          cgroup         cgroup     rw,nosuid,nodev,noexec,relatime,freezer
│ │ ├─/sys/fs/cgroup/devices          cgroup         cgroup     rw,nosuid,nodev,noexec,relatime,devices
│ │ ├─/sys/fs/cgroup/cpu,cpuacct      cgroup         cgroup     rw,nosuid,nodev,noexec,relatime,cpu,cpuacct
│ │ ├─/sys/fs/cgroup/pids             cgroup         cgroup     rw,nosuid,nodev,noexec,relatime,pids
│ │ ├─/sys/fs/cgroup/blkio            cgroup         cgroup     rw,nosuid,nodev,noexec,relatime,blkio
│ │ ├─/sys/fs/cgroup/memory           cgroup         cgroup     rw,nosuid,nodev,noexec,relatime,memory
│ │ └─/sys/fs/cgroup/perf_event       cgroup         cgroup     rw,nosuid,nodev,noexec,relatime,perf_event
```

**Comment by mbiebl** [comment]
> Hmm, can't make sense of `v239-2818-g66e383401`... There is not commit g66e383401 in our tree...

Btw, that's the output of ```git describe``` . You need to strip of the leading 'g' from g66e383401

**Comment by poettering** [comment]
Fix waiting in #11009.
